app-dashboard/js/request.js

```diff
@@ -32,7 +32,6 @@ define([
             return $.ajax({
                 url: url,
                 data: data,
-                dataType: 'json',
```

Comment:
Removed this line to let jQuery determine the content type.
If we set it to JSON, the ajax call will raise error because PostgREST doesn't send JSON body, even though the POST request succeeded.
It is kind of unexpected.